src/views/HomeView.vue

```diff
@@ -1,7 +1,8 @@
 <script setup>
 import { ref } from "vue";
+import Home from "../components/Home.vue";
 
-const tasks = ref({
```

Comment:
Depends on what the backend would return in this situation. I'd assume its a list of tasks that a user can scroll between using the "Skip to Next" button. I'm thinking maybe we want to leave it as tasks and also make it an array of multiple tasks than a single task. Then we also need to model how that skip to next button would work.

Reply:
> I'd assume its a list of tasks that a user can scroll between using the "Skip to Next" button. I'm thinking maybe we want to leave it as tasks and also make it an array of multiple tasks than a single task.

This is how I've been imagining it. A GET request to `api/tasks/` would return a batch of tasks, say 10, where a "batch" is a json object containing an array.

Reply:
Oh I see instead of 10 different requests we bundle them into one request which would lessen the runtime